src/components/views/right_panel/TimelineCard.tsx

```diff
@@ -92,7 +91,7 @@ export default class TimelineCard extends React.Component<IProps, IState> {
     }
 
     public componentDidMount(): void {
-        this.roomStoreToken = RoomViewStore.instance.addListener(this.onRoomViewStoreUpdate);
+        RoomViewStore.instance.addListener(UPDATE_EVENT, this.onRoomViewStoreUpdate);
```

Comment:
Minor: can we use on/addListener consistently? It's `on` above and I think that's our usual.

Reply:
Sure, I just kept with `addListener` because that's what it was using previously.

Reply:
TimelineCard doesn't use `.on` anywhere afaict, so not going to change this.